cmd/influxd/run.go

```diff
@@ -176,7 +176,7 @@ func (cmd *RunCommand) Open(config *Config, join string) (*messaging.Broker, *in
 	// have it occasionally tell a data node in the cluster to run continuous queries
```

Comment:
I think this block should actually be moved into the block at line 344. This whole `RunCommand` needs to be refactored anyway, so we could just punt for now.